R/create_files.R

```diff
@@ -7,57 +7,15 @@
 #'
```

Comment:
The descriptions of these parameters are unclear to me, in particular the `ws_dat` parameter. Is it a dataset, a list, another type of object? It may be useful to describe the functions used to create these objects.
In addition, I would consider renaming the arguments. `meta_fld` can be `meta_url` or simply `url` if what is needed is a URL string. Consider also adding a default URL or an example to make it clear to a user what type of URL is needed here (e.g. is it a URL on sharepoint, github, what is "the meta folder", and how far into this folder should the URL point?).